target/linux/ramips/dts/mt7621_dlink_dir-882-a1.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_dlink_dir-8xx-a1.dtsi"
+
+/ {
+	compatible = "dlink,dir-882-a1", "mediatek,mt7621-soc";
+	model = "D-Link DIR-882 A1";
+
+	leds {
```

Comment:
Just use `leds: leds {` in the DTSI and the reference the label here as you do with the individual leds. Despite, there is no need for the settings the compatible again here.